Core/Core/Logging/SpeckleLog.cs

```diff
@@ -271,23 +292,16 @@ private static void AddUserIdToGlobalContextFromDefaultAccount()
     }
```

Comment:
Note that, as far as I'm aware, this part of the code is not the same ID we're computing in `Account`. @JR-Morgan not sure what to make of this as "it has been working", but seems like a potential for missmatch.

Reply:
Just realised this was supposed to be a multi-line comment but it's not... unhelpful

Reply:
Resolving in favour of https://spockle.atlassian.net/browse/CNX-8750